.commitlintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": [
+    "@commitlint/config-conventional"
+  ],
+  "rules": {
+    "scope-enum": [
+      2,
+      "always",
+      [
+        "android",
```

Comment:
@jyyi1 should also review these scopes - any suggestions? This will ensure we're consistently calling our scopes by the same name.